about.html

```diff
@@ -0,0 +1,135 @@
+          <li class="link"><a href="#">Sponsers</a></li>
+          <li class="link"><a href="#">News</a></li>
+
+        </ul>
+        <div class="header-button"><button type="button" class="button-header">G Gathering</button></div>
+      </div>
+    </nav>
+  </section>
+  <div id="mySidenav" class="sidenav">
+    <a href="#" class="closebtn" onclick="closeNav()">&times;</a>
+    <a href="about.html">About</a>
+    <a href="#">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsers</a>
+    <a href="#">News</a>
+  </div>
+  <section class="about" id="about">
+    <div class="main">
+      <div class="hamburger" onclick="openNav()"><img src="./images/Ham.jpg" alt="hamburger"></div>
+
+      <div class="content">
+        <h1>"Welcome! to the Marketing world"</h1>
+        <h2>A GLOBAL GATHERING <br>OF BRAND BUILDERS & MARKETERS</h2>
+        <div class="inner-content">
+          <p>The Marketing Conference spotlights the latest innovations and breakthroughs in the marketing and advertising industry. Connect with the global community and join thought leaders and storytellers as they explore visionary ideas that engage and
+            inspire
+          </p>
+        </div>
+        <div class="inner-content-bottom">
+          <p>Please contact as per Email for any further questions about global Gathering</p>
+          <h3>globalgather@ggather.org</h3>
+          <div class="under-line"> </div>
+
+        </div>
+      </div>
+
+    </div>
+    <div class="featured-speakers">
+      <div>
+        <h2>About Marketing Conference That’s Bringing This Event To Life!</h2>
+        <div class="under-line"> </div>
+      </div>
+      <div>
+        <p>Marketing Conference is a premier marketing and advertising conference that will draw eminent leaders of the community from all over the world. <br><span class="content-hide">it aspires to help businesses connect with their customers better, enhance their visibility, and build their brand by learning from the best marketers. <br> The conference will unveil the latest technologies, research, and ideas that sell in the realms of content creation, branding, marketing, and advertising.industry. </span></p>
+      </div>
+      <div class="about-content-logo"><img src="./images/new-logo.png" alt="about logo" /></div>
+
+
+    </div>
+    <hr>
+    <div class="featured-speakers">
+      <div>
+        <h2>See the coming Global Marketing Conferences</h2>
+        <div class="under-line"> </div>
+      </div>
+      <div>
+        <p>Take a look at the given two global Marketing Conferences which are taking place in USA</p>
+      </div>
+      <div class="summits">
+
+        <div class="summit-img"><img src=".images/img_banner_agenda_d.png" alt="summit 2022"></div>
+        <div class="over-blur">USA 17 Date: 18th - 19th March, 2022 Venue:Festival City, USA</div>
+        <div class="summit-img"><img src="./images/img_banner_agenda_u.png" alt="summit 2022"></div>
+
+
+
+
+      </div>
+    </div>
```

Comment:
Kindly try to match the `Behance` design for the past section in the mobile version on the about page. the `font size` needs to be smaller and not all images loaded.
| Your design | Behance design |
| --- | --- |
| ![screencapture-raw-githack-monika-soam-Marketing-Conference-About-Page-about-html-2022-03-10-00_04_53](https://user-images.githubusercontent.com/24703171/157548968-b8136624-8519-4349-b44b-dbb4c35fce9e.png) | ![79a95029845175 560699a4328c5 - Copy - Copy](https://user-images.githubusercontent.com/24703171/157548911-bae94ca9-3771-49e7-8712-eb37b3a24670.jpg) |